store/remote.go

```diff
@@ -0,0 +1,53 @@
+package store
+
+import (
+	"time"
+
+	"github.com/gluster/glusterd2/pkg/elasticetcd"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/coreos/etcd/clientv3"
+	"github.com/coreos/etcd/clientv3/concurrency"
+)
+
+func newRemoteStore(conf *Config) (*GDStore, error) {
+	if len(conf.Endpoints) == 1 && conf.Endpoints[1] == elasticetcd.DefaultEndpoint {
```

Comment:
I'm not sure if we'd have a lot of users trying to do this. I expect most users to either use the embedded elastic etcd or use the already available etcd clusters in kubernetes sort of deployments.

That doesn't mean we cannot support this.

Right now I need to ensure that the user has provided a proper endpoint, when GD2 is asked not to start the embedded store. This is the only way (I can think of right now), to check if the user has given proper endpoints that aren't the default. If not, we end up trying to connect to the default, and just stall for a long time (changing the timeout below will help though).